### ISSUE TITLE
Rstudio Markdown editor

### ISSUE THREAD

**Issue by aleruete** [r-lib/pkgdown#1634]
In the newest Rstudio update there is an option to visually edit Rmarkdown files, and it adds this to the beginning
---
editor_options: 
  markdown: 
    wrap: 72
---

This is shown in the HTML produced

**Comment by maelle** [comment]
@aleruete I updated RStudio yesterday so am using the "Juliet Rose" :rose: version and am using pkgdown dev version.

I've been able to add the YAML you mention by using the visual R Markdown editor. However when I build the article, these options are not shown in the HTML produced. Could you please try again and report back?

**Comment by aleruete** [comment]
@maelle will do after holidays! Thanks for checking!

**Comment by aleruete** [comment]
Yes, sorry, I should have closed myself. I can't reproduce the error either.
